library/build.gradle

```diff
@@ -34,6 +35,9 @@ android {
         installOptions "-d", "-t"
     }
 
+    defaultConfig {
+        testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
```

Comment:
There already is a `defaultConfig` section in this file. Can you merge it with the existing one?

Reply:
Removed.